docker-hub/repos/configure/index.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Configuring repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, creating, deleting, consolidating
+title: Configure
```

Comment:
Similar to another comment in the TOC. The tense for the `Configure` title is inconsistent with the TOC entry.